addons/website/static/tests/tours/website_form_editor.js

```diff
@@ -314,6 +323,21 @@ odoo.define('website.tour.form_editor', function (require) {
         {
             content:  "Wait reloading...",
             trigger:  "html:not(:has(#completlyloaded)) div",
+        },
+        {
+            content: 'Verify value attribute and property',
+            trigger: '.s_website_form_field:eq(0) input',
+            run: function () {
+                const inputEl = document.querySelector('.s_website_form_field input');
+                if (inputEl.getAttribute('value') !== 'John Smith') {
+                    console.error('The value attribute is not correctly set');
+                }
+                setTimeout(function () { // Wait for the auto-fillable fields to be filled.
+                    if (inputEl.value !== 'Mitchell Admin') {
+                        console.error('The value property is not correctly set');
+                    }
+                }, 1000);
+            },
```

Comment:
This step is the last of your tour. If you add a setTimeout inside... well it will never be considered as the tour will be marked as finished before reaching the setTimeout code (or if not, it's really by chance). Did you actually try to break the prefill system to see if your tour was indeed red in that case? 

Also: `const inputEl = document.querySelector('.s_website_form_field input');`is not it the `trigger` you declared? You don't need to research it again, you can use `this.$anchor` or something like that.

Anyways, this whole run can actually be replaced by a better trigger which do not need a setTimeout (which would slow down runbot builds):
`trigger: '.s_website_form_field:eq(0) input[value="John Smith"]:propValue("Mitchell Admin")'`
-> That way that step will be marked as ok as soon as an input with the John Smith default value but the Mitchell Admin actual value is found. (propValue is something of ours). Double check what I say but it should work. I have a slight worry that the prefill done in JS might not trigger a step check in the tour system but that would need to be fixed in that case.